README.md

```diff
@@ -85,217 +81,116 @@ If you haven't done so already, at this point go ahead and clone this repository
 
 Let's run the `Main` app with the `register` argument, which does two things:
 
-1. Registers a `TaskDef` named `greet` with LittleHorse.
-2. Registers a `WfSpec` named `quickstart` with LittleHorse.
+1. Registers a `TaskDef` named `verify-identity` with LittleHorse.
+2. Registers a `TaskDef` named `notify-customer-verified` with LittleHorse.
+3. Registers a `TaskDef` named `notify-customer-not-verified` with LittleHorse.
+4. Registers a `WfSpec` named `quickstart` with LittleHorse.
```

Comment:
with LittleHorse server.

It's not abundantly clear to new users that "LittleHorse" is the server.